app/views/taxons/show.html.erb

```diff
@@ -16,17 +16,17 @@
       <div class="grid-row">
         <div class="column-two-thirds">
           <h2 class="taxon-page__section-heading"><%= section[:title] %></h2>
-          <%= render 'govuk_publishing_components/components/document_list',
-            items: section[:documents],
-            margin_top: true,
-            margin_bottom: true
+        </div>
+      </div>
+          <%= render(
```

Comment:
The indentation is off here - this is now inside the `taxon-page__section-group` div rather than being nested inside the `grid-row` and `column-two-thirds`

Reply:
@vanitabarrett Fixed and fixed-up! Thanks!